.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -24,6 +24,10 @@ labels: enhancement
 
 * aws_XXXXX
 
+### Is the affected resource new or existing
```

Comment:
This question may be confusing in the scenario where the feature request is for functionality that requires new resources and also affects existing ones. Is there something we can do with the existing question above to better attempt to get this information?